lib/improver/utilities/filename.py

```diff
@@ -45,6 +45,10 @@ def generate_file_name(cube):
         cube (iris.cube.Cube):
             Cube containing nowcast data
 
+    Kwargs:
+        parameter (str):
+            Optional parameter name to use
```

Comment:
Optional parameter name to use in the output filename rather than taking it from the name of the cube?

Reply:
Agreed.

Reply:
Done.